### PR TITLE
Update for the LumiPCC module veto list and afterglow factors

### PR DESCRIPTION
**Pull request by radla118**
#### PR description:
Update of the LumiPCC module veto list and afterglow parameters implemented in the corresponding configuration files. The workflow remains unchanged.

#### PR validation:
Validated by running runTheMatrix.py -l 1020.0  configuration.

Backport will be requested later.

### CODE REVIEW

**Comment by francescobrivio** [comment]
@cmsbuild please test

File: Calibration/LumiAlCaRecoProducers/python/alcaRawPCCProducer_cfi.py [hunk 1]
```diff
@@ -14,4 +14,4 @@
...
...53003524,353006596,353007620,353010692,353011716,353014788,353015812,353031172,353032196,353035268,353036292,353039364,353040388,353043460,353044484,353052676,353055748,353060868,353063940,353064964,353068036,353069060,353072132,353073156,353076228,353077252,353113092,353114116,353117188,353118212,353122308,353126404,353130500,353133572,353137668,353138692,353141764,353149956,353150980,353154052,353155076,353158148,353162244,353166340,353167364,353170436,353171460,353174532,353175556,353178628,353182724,353186820,353190916,353195012,353196036,353199108,353200132,353203204,353204228,353207300,353208324,353215492,353219588,353223684,353224708,353227780,353228804,353231876,353232900,353235972,353236996,353240068,353241092,353244164,353245188,353248260,353249284,353252356,353253380,353256452,353257476,353260548,353261572,353264644,353265668,353268740,353269764,353272836,353273860,353276932,353277956,353281028,353282052,353285124,353286148,353289220,353290244,353294340,353297412,353298436,353301508,353302532,353305604,353306628,353309700,353310724,353313796,353314820,353317892,353318916,353321988,353323012,353326084,353327108,353330180,353331204,353334276,353335300,353338372,353339396])
```
Comment (mmusich) [inline Calibration/LumiAlCaRecoProducers/python/alcaRawPCCProducer_cfi.py:17]: How is this list determined?

Comment (radla118) [reply Calibration/LumiAlCaRecoProducers/python/alcaRawPCCProducer_cfi.py:17]: The collected Lumi PCC data is reprocessed at the end of each year to exclude the misbehaving modules (the list is obtained from the analysis of the 2022 data). The update is just providing us more precise prompt lumi measurements for 2023.

**Comment by tvami** [comment]
+alca
 * privately also agreed with the developers that this list should be provided dynamically through the DB

**Comment by tvami** [comment]
@radla118 please feel free to prepare the backport PR too

**Comment by rappoccio** [comment]
+1

**Pull request merged**